parser_library/test/workspace/extension_handling_test.cpp

```diff
@@ -62,7 +62,7 @@ TEST(extension_handling_test, extension_removal)
 
     // test no extensions
     library_local lib5(file_mngr, lib2_loc, { {} }, empty_loc);
-    EXPECT_FALSE(lib5.has_file("MAC"));
+    EXPECT_TRUE(lib5.has_file("MAC"));
```

Comment:
This comment is about lines 47 - 55 above.
I would either erased the line comments or change them from `// file must end...` to `// file can end...`

Reply:
The comment is actually correct - if the list of extensions is provided, macros must match end with one of the specified extensions

Reply:
Right...